src/ontology/source/source.owl

```diff
@@ -35480,6 +35481,14 @@ AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000112> <http://purl.ob
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ERO_0002193> "olfactometer"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ERO_0002193> <http://purl.obolibrary.org/obo/ERO_0000004>)
 SubClassOf(<http://purl.obolibrary.org/obo/ERO_0002193> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/OBI_0000306> <http://purl.obolibrary.org/obo/OBI_0000453>))
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ERO_0002195> "digitial ELISA"^^xsd:string)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ERO_0002195> "An molecular assay based on the detection of single enzyme-linked immunocomplexes on beads that are sealed in arrays of femtoliter wells."^^xsd:string)
```

Comment:
Good catch, that's a typo -- I changed the superclass at the last minute. Can you fix it or do I need to?